samples/test/models.test.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const {BigQuery} = require('@google-cloud/bigquery');
+const {assert} = require('chai');
+const cp = require('child_process');
+const uuid = require('uuid');
+
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+
+const bigquery = new BigQuery();
+
+describe(`Models`, () => {
+  const datasetId = `gcloud_tests_${uuid.v4()}`.replace(/-/gi, '_');
+  const modelId = `gcloud_tests_${uuid.v4()}`.replace(/-/gi, '_');
```

Comment:
Are these two variables exactly the same? If so, should there be something distinguishing which is the dataset and which is the model? If that distinguishing factor happens later, consider reducing this to a single var (unless I'm missing something and they actually are different)

Reply:
LOL yep I see it now. I'm gonna go ahead and blame this one on jetlag to save face 🙈